lib/jazzy/config.rb

```diff
@@ -164,10 +171,10 @@ def expand_path(path)
       command_line: ['-x', '--xcodebuild-arguments arg1,arg2,…argN', Array],
       description: 'Back-compatibility alias for build_tool_arguments.'
 
-    config_attr :sourcekitten_sourcefile,
-      command_line: ['-s', '--sourcekitten-sourcefile filepath1,…filepathN',
+    config_attr :sourcekitten_sourcefiles,
```

Comment:
I didn't rename this earlier because it's not backwards compatible (in the config file).